test/Datadog.Trace.IntegrationTests/Datadog.Trace.IntegrationTests.csproj

```diff
@@ -34,4 +34,8 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Testing" Version="5.0.0" />
     <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="5.0.0" />
   </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="FluentAssertions" Version="5.10.3" />
```

Comment:
Thanks for adding this 👍🏼 This should make our lives easier

Reply:
I actually didn't mean to include this in the final PR 😂 But @kevingosse has added it to all projects in #1303, so will remove it from here if he merges first 🙂

Reply:
We should add `FluentAssertions` to `LICENSE-3rdparty.csv`.

Reply:
Actually, we don't distribute it, so I'm not sure about that.

Reply:
We don't currently list 3rd party testing libraries as far as I can see? e.g. xUnit, Moq etc. Should we add those too, or are they not listed because they're not in the end product components?

Reply:
I think you're right. Never mind!